spec/features/idv/doc_auth/address_step_spec.rb

```diff
@@ -4,15 +4,36 @@
   include IdvStepHelper
   include DocAuthHelper
 
-  context 'with mainland address' do
+  let(:puerto_rico_guidance_text) do
+    t('doc_auth.info.address_guidance_puerto_rico_html').gsub('<br>', "\n")
+  end
+
+  context 'with a mainland address' do
     before do
       sign_in_and_2fa_user
       complete_doc_auth_steps_before_address_step
     end
 
+    it 'renders the Puerto Rico address guidance as hidden text' do
+      guidance = page.find_by_id('puerto-rico-extra-text', visible: false)
+
+      expect(guidance).not_to be_visible
+    end
+
+    context 'that gets changed to a Puerto Rico address' do
+      before do
+        select 'Puerto Rico', from: 'idv_form_state'
+      end
+
+      it 'shows the Puerto Rico address guidance' do
+        guidance = page.find_by_id('puerto-rico-extra-text')
+
+        expect(guidance).to be_visible
+      end
+    end
+
```

Comment:
These JavaScript tests are very slow to run, can we consider combining a few? On a similar note, since a feature test should be responsible for testing the experience as a user would encounter it, I don't think hidden text is something we should explicitly test for. Maybe we have a single test that checks that the content is not present 'til the selection is made?

```suggestion
    it 'renders the Puerto Rico address guidance when Puerto Rico is selected state' do
      pr_html_text = t('doc_auth.info.address_guidance_puerto_rico_html').gsub('<br>', "\n")
      expect(page).not_to have_content(pr_html_text)
      select 'Puerto Rico', from: t('idv.form.state')
      expect(page).to have_content(pr_html_text)
    end
```

Reply:
I'm very uncomfortable with not checking that the text is always rendering, but I can make that a controller test.

Reply:
Combined into one spec.